### PR TITLE
switch to free2z.cash lightwalletd

### PR DESCRIPTION
**Pull request by zancas**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -48,7 +48,7 @@ This will launch the interactive prompt. Type `help` to get a list of commands.
 ## Notes:
 * If you want to run your own server, please see [zingo lightwalletd](https://github.com/zingolabs/lightwalletd), and then run `./zingo-cli --server http://127.0.0.1:9067`
 * The default log file is in `~/.zcash/zingo-wallet.debug.log`. A default wallet is stored in `~/.zcash/zingo-wallet.dat`
-* Currently, the default, hard-coded `lightwalletd` server is https://lwdv3.zecwallet.co:443/. To change this, you can modify the `DEFAULT_SERVER` const in `config/src/lib.rs`
+* Currently, the default, hard-coded `lightwalletd` server is https://zuul.free2z.cash:9067/. To change this, you can modify the `DEFAULT_SERVER` const in `config/src/lib.rs`
```
Comment (skyl) [inline README.md:51]: mainnet.lightwalletd.com should on paper be the most reliable but it hasn't been recently. I think being able to configure and easily switch (ENV variable?) should be high priority feature.

File: config/src/lib.rs [hunk 2]
```diff
@@ -21,7 +21,7 @@ use zcash_primitives::{
     constants,
 };
 
-pub const DEFAULT_SERVER: &str = "https://lwdv3.zecwallet.co";
```
Comment (skyl) [inline config/src/lib.rs:24]: Should be easy to make an env variable with default?

**Comment by dannasessha** [comment]
ACK

**Review by dannasessha — APPROVED** [review]
I think this is a fine change in a pre-production state!

Comment could lead to more conversation, and I have reached out for confirmation of the infra, so this is a planned-merge approval - but not a merge yet.

File: README.md (hunk 1 above)
Comment (zancas) [reply README.md:51]: This seems sufficient to me.   I am +1 on a merge.  I cannot merge, since I opened it.   Can someone please merge?

Comment (juanky201271) [reply README.md:51]: Nope, no way to change the server in the UI mobile

**Comment by dannasessha** [comment]
Very fast sync!!

This looks great.

**Pull request merged**